### PR TITLE
Notification sentences

### PR DESCRIPTION
**Pull request by zwim**
On Notifications when `getOptionText` returns not a value, use colon in message.
Reworded some notifications with "on" "off".
See https://github.com/koreader/koreader/pull/7718#discussion_r638670551

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7759)

### CODE REVIEW

File: frontend/apps/reader/modules/readertypeset.lua [hunk 1]
```diff
@@ -138,19 +138,27 @@ end
...
+    else
+        Notification:notify(_("Embedded styles are off."))
+    end
     return true
 end
 
 function ReaderTypeset:onToggleEmbeddedFonts(toggle)
     self:toggleEmbeddedFonts(toggle)
-    Notification:notify(T( _("Embedded fonts are %1."), optionsutil:getOptionText("ToggleEmbeddedFonts", toggle)))
+    if notification then
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readertypeset.lua:151]: toggle, I imagine ;-)

**Review by Frenzie — COMMENTED** [review]
Cool, thanks. Just the one little thing.

File: frontend/apps/reader/modules/readertypeset.lua [hunk 2]
```diff
@@ -138,19 +138,27 @@ end
 
 function ReaderTypeset:onToggleEmbeddedStyleSheet(toggle)
     self:toggleEmbeddedStyleSheet(toggle)
-    Notification:notify(T( _("Embedded styles are %1."), optionsutil:getOptionText("ToggleEmbeddedStyleSheet", toggle)))
+    if toggle then
+        Notification:notify(_("Embedded styles are on."))
+    else
+        Notification:notify(_("Embedded styles are off."))
```
[the remark this answers is not in the document]
Comment (NiLuJe) [reply frontend/apps/reader/modules/readertypeset.lua:144]: Or even clearer the other way around: `Enabled <stuff>` / `Disabled <stuff>`.

[the remark this answers is not in the document]
Comment (Frenzie) [reply frontend/apps/reader/modules/readertypeset.lua:144]: Agreed. Please put it in a PR. ;-)

**Pull request merged**